src/native/containers/dn-simdhash-utils.c

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "dn-utils.h"
+#include "dn-simdhash-utils.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+// Define a default implementation of the assert failure function.
```

Comment:
Should we put something like this in minipal? it seems generally useful

Reply:
I think I'd like to see more places where we'd use it before I move it to minipal. We can extract it the next time we'd have a use for it.